server/starAPI.js

```diff
@@ -0,0 +1,131 @@
+import { Router } from "express";
+import logger from "./utils/logger";
+import db from "./db";
+const router = Router();
+
+// Getting all star from the database
+
+router.get("/stars", async (req, res) => {
+	try {
+		const result = await db.query("SELECT * FROM stars");
+		res.json(result.rows);
+	} catch (error) {
+		logger.error(error);
+		res.status(200).json(error);
```

Comment:
You shouldn't be returning status 200 when there has been an error.
In this case, it should only happen if the database is down or something like that, so 500 is probably the one to use.

Reply:
Not a big enough issue to hold up the merge.